### PR TITLE
Create features.md

### PR DESCRIPTION
**Pull request by ybolognini**
Doc to list features mobile/desktop

### CODE REVIEW

**Comment by ybolognini** [comment]
@sbrunner Please review

**Comment by sbrunner** [comment]
Travis don't like trailing whitespace...
https://travis-ci.org/camptocamp/ngeo/jobs/293201799#L656-L685

File: docs/features.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Ngeo Features
+
+This document lists ngeo's features and if they are visible/configurable in the mobile interface.
+
+Feature | Sub-feature | Desktop | Mobile
+------- | ----------- | ------- | ------
+Navigation | Zoom&pan | X | X
+. | Map rotation | (2.3) | (2.3)
+. | Geolocation | X | X
+. | Navigation bar | X | -
```
Comment (sbrunner) [inline docs/features.md]: => Info bar

File: docs/features.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+. | Themes (non-flush) | X | (2.3)
+Layers | (Un)select layer/group | X | X
+. | Clear all groups | X | X
+. | Remove group | X | X
+. | Reorder groups | X | -
+. | Legend | X | X
+. | Go to scale | X | X
+. | Info on layer | X | X
+. | Opacity | X | (2.3)
+. | Time layers | X | X
```
Comment (sbrunner) [inline docs/features.md]: Not in mobile

File: docs/features.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Query | Point query | X | X
+. | Rectangular query | X | X
```
Comment (sbrunner) [inline docs/features.md]: Not in mobile

File: docs/features.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+. | Results window | X | X
+. | Results window filter | X | -
+. | Results grid | X | X
```
Comment (sbrunner) [inline docs/features.md]: Not in mobile

File: docs/features.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+. | Results grid zoom to | X | -
+. | Results grid export | X | -
+. | Infos on point (includes raster) | X | -
```
Comment (sbrunner) [inline docs/features.md]: Mobile, throw measure point

File: docs/features.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+StreetView | StreetView panel | X | -
+External layers | Online WMS | (2.3) | -
+. | Online WMTS | (2.3) | -
+. | Local KML | (2.3) | -
+. | Local GPX | (2.3) | -
+. | Query point on WMS | (2.3) | -
+Permalink | Share window | X | -
+. | Location | X | X
+. | Layers/groups/themes | X | X
+. | Drawing | X | (2.4?)
```
Comment (sbrunner) [inline docs/features.md]: mobile 2.4 read only

File: docs/features.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+. | CSV export | X | -
+StreetView | StreetView panel | X | -
+External layers | Online WMS | (2.3) | -
+. | Online WMTS | (2.3) | -
+. | Local KML | (2.3) | -
+. | Local GPX | (2.3) | -
+. | Query point on WMS | (2.3) | -
+Permalink | Share window | X | -
+. | Location | X | X
+. | Layers/groups/themes | X | X
```
Comment (sbrunner) [inline docs/features.md]: Background

File: docs/features.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+. | Snapping | X | -
+Profile | Draw profile | X | -
+. | CSV export | X | -
+StreetView | StreetView panel | X | -
+External layers | Online WMS | (2.3) | -
+. | Online WMTS | (2.3) | -
+. | Local KML | (2.3) | -
+. | Local GPX | (2.3) | -
+. | Query point on WMS | (2.3) | -
+Permalink | Share window | X | -
```
Comment (sbrunner) [inline docs/features.md]: MapCrosshair, MapTooltip, Dimensions, WfsPermalink, External layers (2.3)

File: docs/features.md (hunk 6 above)
Comment (ybolognini) [reply docs/features.md]: Also to add: Object Editing features (viewer+editor). See with ELU

**Pull request merged**